infra/module-swapper/cmd/swap.go

```diff
@@ -54,28 +66,30 @@ func getModuleNameRegistry(dir string) (string, string, error) {
 	if err != nil {
 		return "", "", err
 	}
-
-	// GH remote will be of form https://github.com/ModuleRegistry/ModuleName
-	if !strings.Contains(remote, "https://github.com/") {
-		return "", "", fmt.Errorf("Expected GitHub remote of form https://github.com/ModuleRegistry/ModuleRepo. Got: %s", remote)
+	u, err := giturl.Parse(remote)
+	if err != nil {
+		return "", "", err
+	}
+	if u.Host != "github.com" {
+		return "", "", fmt.Errorf("expected GitHub remote, got: %s", remote)
 	}
+	orgRepo := u.Path
+	orgRepo = trimAnySuffixes(orgRepo, []string{"/", ".git"})
+	orgRepo = strings.TrimPrefix(orgRepo, "/")
 
-	// remotes maybe suffixed with a trailing / or .git
-	remote = trimAnySuffixes(remote, []string{"/", ".git"})
-	namePrefix := strings.ReplaceAll(remote, "https://github.com/", "")
-	if !strings.Contains(namePrefix, "/") {
-		return "", "", fmt.Errorf("Expected GitHub org/owner of form ModuleRegistry/ModuleRepo. Got: %s", namePrefix)
+	split := strings.Split(orgRepo, "/")
+	if len(split) != 2 {
+		return "", "", fmt.Errorf("expected GitHub remote of form https://github.com/ModuleRegistry/ModuleRepo, got: %s", remote)
 	}
-	moduleRegistry := namePrefix[:strings.LastIndex(namePrefix, "/")]
-	repoName := namePrefix[strings.LastIndex(namePrefix, "/")+1:]
+	org, repoName := split[0], split[1]
```

Comment:
Would it be valuable to include a check on valid `org` values?  e.g. `GoogleCloudPlatform` `terraform-google-modules`?

Reply:
Hmm I had added support for a `moduleRegistryPrefix` so users could explicitly specify a different org instead. Did we have a usecase were restricting is better?

Reply:
Ahh - I see your plan.  I was thinking it might be valuable to include those orgs as the "defaults" and permit overriding with the flag, but this direction is also fine.